### PR TITLE
Teacher Application: Scholarship question changes

### PR DESCRIPTION
**Pull request by islemaster**
image

([PLC-104](https://codedotorg.atlassian.net/browse/PLC-104), [spec](https://docs.google.com/document/d/1iubRcgyT8M1o9VU5n-T8-IE9GsmRAsT7EN1pWtVEWfc/edit#bookmark=id.hvfd7drh8ib6))

Makes some requested changes to this question:
- Changed question text
- Removed two options
- Question appears (and is required) only if you've been matched to a regional partner.

### CODE REVIEW

File: dashboard/app/models/pd/application/teacher1920_application.rb [hunk 1]
```diff
@@ -263,35 +260,33 @@ def self.required_fields
...
         if hash[:pay_fee] == TEXT_FIELDS[:no_pay_fee_1920]
-          required.concat [:scholarship_reasons]
+          required << :scholarship_reasons
         end
 
         if hash[:program] == PROGRAMS[:csd]
-          required.concat [
-            :csd_which_grades,
-          ]
+          required.concat << :csd_which_grades
```
Comment (clareconstantine) [inline dashboard/app/models/pd/application/teacher1920_application.rb]: don't need the `.concat`

**Comment by hacodeorg** [comment]
Not in your PR but should we change the question in your screenshot to _Will ~you or~ your school be able to pay fee?_

**Review by hacodeorg — APPROVED** [review]

**Comment by islemaster** [comment]
Nice catch. Inquiring with stakeholders now...

**Review by clareconstantine — COMMENTED** [review]
one tiny thing, looks good otherwise!

**Review by clareconstantine — APPROVED** [review]

**Pull request merged**